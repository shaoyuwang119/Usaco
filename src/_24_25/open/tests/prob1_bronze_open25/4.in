90 94
D
LD
DLD
LDWD
DLLLD
WDWDWD
WWWDWWD
LDWDDWDD
DLDWLWWLD
DWWDLLLDDD
LLLWDDWDLLD
DLWDDWWWDWWD
WLDLLDWWDDLWD
LLWLDWWLWLDDWD
DWLLDWDDLLDDLDD
WWDLDLDDLLWWDWDD
DWDWLLLLLWWDLDLLD
WDLWWWLLDDDLDDWLWD
DDLLLWDLLDLWWWDWDDD
DWWWWWWDDDLLDWLLDDWD
WLLLWDDDWWLLDWDDDLDWD
DLWDWLDDDDDLLLWWDWLWDD
WDLDLDLDWDWDLDLWDWWWLLD
WLWLWLWWDDLWDWDWLDDWLLLD
LDLWLWWDLLDDDLLDDWWLDWDLD
WDDDLWWDDDLWDLWLWWLWDDWDLD
WDDLWLDLDWWDDDWDWDLWDWWDDDD
DLDWWWDDDDLDDDLDDDWLWWWWWLLD
DDWLLLDDDLDWDWLDWDLDWLDWDWDWD
LWDDWLDLLLDLWDLDWWLWDLWDLLDLDD
WLWWDWDLWWDWWLDWWLWDWWWDWWDDLWD
DLLLWWWWWLDLDLWWLWDDDLLWWWLLLLLD
WLWWDLLDDDLWWWWLLWDWWDLWLWWDWWDLD
LDWLDDLDDWDDDDWDDWDLWDWDWDLLLLDDLD
DDDWDDLLLLLDLDDLLDWLDWWWDLDDLLLLWWD
LLLWDLWLWLDLLWLWLWDLDLWWWDLWLDWLWWDD
DLWLDDWWWDDWLLLWWWLDWLWDWLLLLLLDDWWLD
WDWWWWLLLDLDLWWDDLWWDWLWWWWDWDDDWWLLLD
WDLLDWDWDDLDDLDLLDLLDLWDWDLDDLWDWLDWWDD
WWDWWWWDDWLWLDDWDLWDWDDWDWLDDDWWLLWLLLWD
WDDDDLWWLWWWDWWDLDLDWDWDLDDDDLLLDDWLWWLDD
WLWDLLDDWLLWLLLDLLWDDWWWDLLLDLDLDLWWWLLLWD
WDDWWLDWWDDLWLLWWDWDWWLDDDLLDLWLDDWDWWDLLDD
DLWWDDDLWLWWWLLLWWWLWLLDWWWDDLLWLLWWLDWWLWDD
WWLDWLDWWLDDDDWWDDWLLWLLLLWDWWDDWWWLLWDLDDDLD
LLLLDWDLLDDWDDDLDWLDLWWDLLWDLWWLLDLDLWDWLDDWLD
LDLWLWWLDDDDLWDWDLWDLDLLDLDLWLLDDWLDLDLDWDDDDLD
WLDLWWLWLLLLLWDWLDLWLLWWDWDLDDLLDLDWDWLLWDWDLWLD
DLLWDLWWLWWDLLWWDLLLDLLLLLDWLLDDWDLWWWWWLLWDDWLWD
DDDDWDWWWWLLDWWWWDWWWDDWLWWWLDWWLLLDLLLWLLWLWWWWWD
DLLLLLLLWDDDWLDLWLDWLDLWDDLLWWDWWWDWLDLDDLWWDWWDDWD
DLWWLLLWWLLWLWWLLWDLWWWLDWWLLWDLWDDDLDDWDLDWWDWWDLLD
DLLWLDLDDWWLLDDWLWLLDLWLLLLLDDDDWDDWLDWLLDLDDLDLWLDLD
DDDWLWDDLLLWLDWDWWWDDDDWWWDLLWLWDWDDLLWLDWLDWLWWLWWWDD
LWDDWDDDDWLDDWWLLLDDDWLLDLWLDDLLDLWWWWDDLLDDLWDLWWWDDLD
WWLDDWWWLDWDWDDDLWDLLWLWDWDDWWWDLLLWDDWWLWWWLLWDDLWDDDDD
WWWLWLWLLLDLWWWDWLWWLLDLWWLWLWLDLDLDLLLLDWDWDWDWDDWDWDLLD
DDWDWLLLLLDWDDWWLWLWDDLWLDDDWWLWWWLWDDDDWDWDLWDDDWLDLDLDWD
DLWDDDWDDDWWWDDDWDWDWDWWWWLDWLLDLLLLWDLWLWDDDWDDWDLWLDWDDLD
LWDWLWLLWWWWWLLWDWLDDDLDDLWLWLLDDLDWLWWLLLWWDWLDWDWWLDLWWWWD
LLDLWLLLWWDDWDWWLWWLLDLLDDLLWDWDDWLDWLWDLDWLWWDLLDLWWWDLLLWWD
DWLDLDWLWWWWLDLLWDLDLWDLDWWLDDDDWLDWLDWLDLWWDWWDWLWLDDLLDDLWDD
DLLDWLLLLDWLWDDLLDWLWDWWLDDWWLDDDDDLDWWWWWDWWLDLWDWLLWLWDLWLWWD
WWWWDDDLDLDDDWDWDLLDLLLWDDWWLDLWDLWDDLLDLLLWDWDWDLDDWWWLLLLLDWWD
DWWDLLWWDLLDLWLWDLDDDDLWDDLDLDLWLWLDWLLDWLDDLWDDDWLWDWWLDLDWLWWLD
WWWLLDWDLDWWWDWLLLLLWWLLWDWWWWWDLLDDWDDDDDWLWWLWDDDWLDDLWWDDLDDLWD
WLLLDDDLDWLDLDWLLDDWDDDDDLDDDWDLWWDWLDLLWDDLWDWWDWWLDLWDLDDWDLWLDLD
DLLWDWDDLLDWWWLDLWWDWWDLWLDWDDDLDLLDLDLWWWDDWLLDLWDDDDLWLLDDWLDLDWLD
DLDLLWWLLDDDDLWWWLWLLLWWLDLDDDLLLLDDDLDDWLWWWDLLLDDDWWLDLLDWLLDLLWLDD
DWLWLLLLDWLLLLDDLLDWDLDLLWWWWWWLLLWLLDWWWWDLDDWDLLLLWLLLLWDDWDDDWDLDDD
DDDLDWWWWDLLWLDLDLLWDWDLWLLDWDWWDDLLDLWDDWWWDLDWDDLDWWWLWWDDWLWWDWWWLDD
WDDLWDDDDWDWLDLLDDDDWWLLWLDDLDDDWDWLWDDWLLLDLWDWWWLDLWDLDWWLDDLWWDWLWLDD
WDDLWLWLLLWWLWDDWWDDLLLDLLLWDDLWLDWLDWDLLLLWWDLWDLWLWLWDDLDLLLDLLDLWWDLLD
DLWDLDLWLWWDLLLWDWLDWLDDDLLDLWLLLWLWLLWDWLDWDWDWWWDDDDDDLWLLWDWDWLLDWDWWDD
DLDLLDWWDLDWWDLDWLDDDLLDLLWWWDLWWLLLWLLWDWLWDLWLWWDLLWWWDLWLWLLDLDLDDDLDWDD
WLWDWLWLDLLWDWDLDDWDLDDWWLLLLWWDWDWDDDWDWDWDDDLDLWLWWLDDWLLWLDDLDDWLWLDWWLWD
DLLDDLDLDDLDWLLLDWLLDLWWDLDWLDWWDDWWDWWDDLLDWLLDDLWWWDDDLDWDLDWDWWLDDLDWWWDDD
WWLLDWLDDLDDLLLDDWWLWWDWWWWLLDLDLWDDDWWDDWWDWLDWLDLWWDDDWWDLDDWWLWLLLWDLLWLDWD
LWLLLLLLDDLWDWWLWDWWDLLDLDLLLDLWWDDLLWLDDDLDDWWDWDWDWLWDDWLLWLWWDLLWWWWLLLLWWLD
LWLDWWWLDLLLWDDDLDLDDDDDLWDLDDDWLWDWDLDDLWWLDDWDWLLWLLLDLWLDLWDDDDWDLWLLWDDLLLDD
LLWLLWWWDWWDDLWWDDLDLWWDDLWWDLLWWDWWWDWLLWWDLDWWWLDWDDWWLDWLWWDDWWDLDDLDLDDWDLLLD
DWLDDLLWWLLLWLLDWDDWWLLWWLLDLLDWLWLDWWWWDDLLDLWLWLDLWDWWLDDLLLLWLLWWDWDLWWDLWDWWDD
WLLLLDWLWDLDLDDLLDLLLDWWLWDDDLLWWWDDWLLWDDDDLWLLWWWDLLDLDDLDDLWDDDDWDLWWWDDDDWDLWWD
LWWDDWLWLWWWDDLDDLWWDWLDWWWWWWDLDLLWWWDDWLWDDLLDDDDDWDLWDDWDDDDDDWLWDWDDWWWDDDDWLLWD
LWWLDDDWWLDDDDWDWWDWWWDDWDLWLWWDLDLWWWDDWLWWLDDDWWWWWDLWDDLDWDDLLLDLDDLLLLWLWDDWWWDLD
DWLWLWDLWWLDLDLWWLDWLDLLLWDLLLDWLLWLDWDDLLLDDLLLDWLDDLDDWWDLLLLDDLWDLWWWLLDLWLDDDLWWDD
WWLLLWDLDLWDLWLLLDLDLLWWLWWDWLDDDLLWWWDWLWLLLWDLWDDDLDDLDLLLWLLWLWLDDWLDDDLWDLLDLWDLDWD
WDLWDWDDLDWWLLLWLLDLLWDWLLDDDWWLWWWLWLLWDWDDWDLLDWWWWDLLLWLLDDDDLWLDDLWLWLWLDWLDLDLDDLWD
WLWWLLLDLWWDWLDWLWLDDLLWDWWDDWLDWLDDLWDDWLWDLWWLWLDLWWDDWLLWLDWLDDDDWDLDLDWLWLDDLWWDLDLLD
DLLDDLWDWDDLLWDLLWDDLWDDDDDDLDDWWLWWDLDDLWLWDLDWWLDDDWLLDWLDLDWLLDDWDDLLWDWLLWWLWLDDWWWLLD
36 16
72 46
60 30
12 90
28 26
83 80
55 72
11 15
36 90
37 87
32 48
21 87
70 85
80 39
71 18
1 13
6 56
31 80
36 78
65 71
51 17
66 81
1 69
85 9
4 22
32 7
80 10
65 75
10 72
6 77
9 44
53 68
35 85
52 15
53 6
64 77
6 21
24 82
23 2
55 81
82 11
63 76
16 31
2 43
45 66
69 43
59 80
73 14
55 81
58 49
70 22
17 67
10 26
20 18
80 70
81 86
29 85
73 14
23 15
79 9
72 84
18 54
3 88
59 9
61 60
88 87
50 45
87 79
41 88
84 72
33 6
17 22
6 86
67 24
8 38
22 19
33 45
28 22
33 16
44 79
19 41
17 78
67 81
42 57
11 19
13 20
28 18
33 45
25 55
90 4
83 45
77 5
63 44
1 12
