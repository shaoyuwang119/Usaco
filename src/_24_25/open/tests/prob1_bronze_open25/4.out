1859
1700
1700
1700
1044
1859
1211
1376
1859
2324
2624
1539
531
2171
1211
1211
1211
1211
1859
1376
2324
1044
2475
704
1539
2016
1539
1700
1044
1700
2171
2324
1700
1539
2475
1859
1211
2475
704
1376
2016
1376
1044
1859
1700
1700
1376
2324
1376
2324
1700
1859
1539
1539
2624
1376
1376
2324
1700
1859
704
875
1539
1044
1044
2324
1211
2475
2171
704
1211
1211
2171
1700
1700
1859
1211
1211
1859
2016
1700
1539
1211
1700
1211
1211
1044
1211
1700
1044
875
1539
2016
2324
